src/evaluate.ts

```diff
@@ -39,10 +39,10 @@ function interruptAllEvaluations() {
     vscode.window.showInformationMessage("Not connected to a REPL server");
 }
 
-function addAsComment(c: number, result: string, codeSelection: vscode.Selection, editor: vscode.TextEditor, selection: vscode.Selection) {
+async function addAsComment(c: number, result: string, codeSelection: vscode.Selection, editor: vscode.TextEditor, selection: vscode.Selection) {
     const indent = `${' '.repeat(c)}`, output = result.replace(/\n\r?$/, "").split(/\n\r?/).join(`\n${indent};;    `), edit = vscode.TextEdit.insert(codeSelection.end, `\n${indent};; => ${output}\n`), wsEdit = new vscode.WorkspaceEdit();
     wsEdit.set(editor.document.uri, [edit]);
-    vscode.workspace.applyEdit(wsEdit).then((_v) => {
+    await vscode.workspace.applyEdit(wsEdit).then(_v => {
```

Comment:
Since you're using `await` here, you could get rid of the `.then` and just add the inner code below the `await`. You probably know that though. It's just a consistency thing in my head. :smile:

Reply:
This is some old code, when I understood even less of promises and await/async than a do today. 😄 I'll fix!